packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -94,7 +99,7 @@ module.exports = {
       //  evmVersion: "byzantium"
       // }
     }
-  },
+  }
```

Comment:
This comma should stay.

Reply:
`prettier` again :(

Reply:
Same, please don't use `--no-verify` flag on commits.

Reply:
So what should you do when committing non-JS files or in situations like this where you don't want prettier to update things? I think in this case using that flag should be fine. Do you disagree or have a different solution?

Reply:
We were discussing that if we can somehow exclude some files from using prettier. I have raised an issue for this https://github.com/trufflesuite/truffle/issues/5360.